src/wasm/wasm-debug.cpp

```diff
@@ -374,9 +385,21 @@ static void updateDebugLines(const Module& wasm,
     // All the addresses we need to write out.
     std::vector<uint32_t> newAddrs;
     std::unordered_map<uint32_t, LineState> newAddrInfo;
+    // If the address was zeroed out, we must ignore the entire range.
```

Comment:
By 'ignore' do you mean we leave them unchanged, or drop them from the output?

Reply:
We drop them entirely. I'll improve the comment.

Reply:
Also the PR title.